src/components/Channel/Header.vue

```diff
@@ -6,7 +6,7 @@
       <i class="icon-menu4"></i></label>
 
     <div class="channel-header">
-      <span class="channel-name" :class="[channel.type]">{{ label(channel) }}
+      <span class="channel-name" :class="[channel.type]" v-html="label(channel, true)">
```

Comment:
this looks like an error to me?
old version inserted label into span without messing with the existig content. 

You are doing v-html that will replace all existing content of that tag:

https://jsfiddle.net/Laq8b60n/1/